packages/database/src/realtime/WebSocketConnection.ts

```diff
@@ -87,7 +87,7 @@ export class WebSocketConnection implements Transport {
    */
   constructor(
     public connId: string,
-    repoInfo: RepoInfo,
+    private repoInfo: RepoInfo,
```

Comment:
I don't think we need a full reference to repoInfo. Can we just grab `nodeAdmin` as a field?

Reply:
You caught me being lazy